docs/source/tools/text_normalization.rst

```diff
@@ -0,0 +1,34 @@
+Text Normalization
+==================
+
+Text normalization system for english, e.g. `123 kg` -> `one hundred twenty three kilograms`
+
+Offers prediction and evaluation on text normalization data, e.g. `Google text normalization dataset <https://www.kaggle.com/richardwilliamsproat/text-normalization-for-english-russian-and-polish>`__ :cite:`tools-norm-ebden2015kestrel`, :cite:`tools-norm-sproat2016rnn`, :cite:`tools-norm-taylor2009text`.
+
+Reaches 81% in sentence accuracy on output-00001-of-00100 of `Google text normalization dataset <https://www.kaggle.com/richardwilliamsproat/text-normalization-for-english-russian-and-polish>`__, 97.4% in token accuracy.
+
+More details could be found in `this tutorial <https://github.com/NVIDIA/NeMo/blob/main/tutorials/tools/Text_Normalization_Tutorial.ipynb>`__.
+
+Example evaluation run:
```

Comment:
Input data format? and the output format?

Reply:
up to @yzhang123 in a separate PR